run.py

```diff
@@ -919,79 +919,86 @@ def alignments(args):
       genomeid_to_taxid = json.load(inf)
 
    for sample in get_samples(args):
-      combined_output_compressed = sample + ".alignments.tsv.gz"
-      if combined_output_compressed in existing_outputs:
-         continue
+      for db in ["human", "hv"]:
+         combined_output_compressed = "%s.%.alignments.tsv.gz" % (
+            sample, db)
+         if combined_output_compressed in existing_outputs:
+            continue
 
-      with tempdir("alignments", sample) as workdir:
-         outputs = []
-         for potential_input in available_inputs:
-            if not potential_input.startswith(sample): continue
-            if ".settings" in potential_input: continue
-            if "discarded" in potential_input: continue
+         with tempdir("%s alignments" % db, sample) as workdir:
+            tmp_outputs = []
+            for potential_input in available_inputs:
+               if not potential_input.startswith(sample): continue
+               if ".settings" in potential_input: continue
+               if "discarded" in potential_input: continue
 
-            output = potential_input.replace(".gz", ".alignments.tsv")
-            inputs = [potential_input]
-            if ".pair1." in output:
-               output = output.replace(".pair1.", ".")
-               inputs.append(potential_input.replace(".pair1.", ".pair2."))
-            elif ".pair2" in output:
-               # We handle pair1 and pair2 together.
-               continue
+               tmp_output = potential_input.replace(".gz", ".alignments.tsv")
+               inputs = [potential_input]
+               if ".pair1." in tmp_output:
+                  tmp_output = tmp_output.replace(".pair1.", ".")
+                  inputs.append(potential_input.replace(".pair1.", ".pair2."))
+               elif ".pair2" in tmp_output:
+                  # We handle pair1 and pair2 together.
+                  continue
 
-            outputs.append(output)
+               tmp_outputs.append(tmp_output)
 
-            # TODO(jefftk): do we have a problem when read1 or read2 is too
-            # short?  I remember Bowtie choking on these before.
+               # TODO(jefftk): do we have a problem when read1 or read2 is too
```

Comment:
Do you already know more here?

Reply:
I think it's fine, no issues yet, but I'll need to run on more data to be sure.